_test/test_tobyfit/test_refine_crystal/test_tobyfit_refine_crystal_1.m

```diff
@@ -68,6 +68,16 @@ function test_tobyfit_refine_crystal_1 (option)
 % File to which to save results of refinement
 savefile='test_tobyfit_refine_crystal_1_out.mat';   % filename where saved results are written
 
+test_tobyfit_dir = fullfile(horace_root(), '_test', 'test_tobyfit');
+addpath(test_tobyfit_dir)
+cleanup = onCleanup(@() rmpath(test_tobyfit_dir));
+
+% Seed the RNG for reproducibility
+seed = 774016;  % This seed provides a passing test at time of writing
```

Comment:
These two comments can be combined. They need duplicating in the two other tests you're using a fixed seed for as well.

I'd suggest renaming the variable as `fixed_seed` to make it explicit.